docs/EPIC_SPECIFICATION_GUIDE.md

```diff
@@ -0,0 +1,37 @@
+# Lisk Hub Epic Specificaiton Guideline
+
+Lisk Hub team will gather community enhancement proposals in the backlog of the project. 
+
+Once a substantial ammount of proposals have been identified as belonging to the same type/category of enhancemnt, an epic will be created to handle the development journey towards it's implementation. 
+
+## Table Of Contents
+
+1. [Requirements](#requirements)
+1. [Epic creation best practices](#epic-creation-best-practices)
+1. [Epic workflow](#epic-workflow)
+
+
+## Requirements
+- [ZenHub](https://www.zenhub.com/)
```

Comment:
It should say more, at least something like: 
`Install ZenHub browser extension to your browser (Chrome and Firefox supported) to be able to access the Epic feature of ZenHub in Github`